### ISSUE TITLE
Does it work?

### ISSUE THREAD

**Issue by emresenyuva** [FirestonnneTFM/TFMBot#1]
this project? does it work if workings please contact me.

**Comment by FirestonnneTFM** [comment]
Yes, it works. The only two caveats as explained in the readme are (1) this only works on POSIX operating systems (I tested on linux) and (2) that it requires keys that are compiled into the client. I'll probably make a new repository for the code that I use to get the keys, since I imagine that people who have the means to do so would have already made their own bots. But everything in here works provided that those two requisites are met.

**Comment by emresenyuva** [comment]
i have keys parser. like connection key, version, crypto key(20),  auth key. but idk c lang :(
parser written in python.
parser output, like this: 
Connection Key: "CnhsuzNvZt", Version: 401
BKey: [25, 53, 52, 46, 43, 37, 46, 23, 56, 23, 67, 111, 106, 112, 113, 127, 72, 113, 82, 125]
function parse_auth_key(arg1) {
   _loc1_ = arg1;
   _loc1_ = _loc1_ ^ 1 << 17;
   _loc1_ = _loc1_ ^ 1 << 19;
   _loc1_ = _loc1_ ^ 2519937;
   _loc1_ = _loc1_ ^ 11853102;
   _loc1_ = _loc1_ ^ 14116348;
   _loc1_ = _loc1_ ^ 1 << 27;
   _loc1_ = _loc1_ ^ 1 << 5;
   _loc1_ = _loc1_ ^ 1 << 21;
   _loc1_ = _loc1_ ^ 1 << 2;
   _loc1_ = _loc1_ ^ 8073195;
   _loc1_ = _loc1_ ^ 2234725;
   _loc1_ = _loc1_ ^ 11215568;
   _loc1_ = _loc1_ ^ 1 << 19;
   return _loc1_;
}
if you are using skype, add me: emresenyuv@hotmail.com
